### DISCUSSION TITLE
Why is does `sveltkit package` copy the package.json?

### DISCUSSION THREAD

**Discussion by zeekrey** [sveltejs/kit#5206] [Q&A]
Hey there!

I just started using Sveltkit to build a Svelte component library. And I was wondering why the `sveltkit package` command copies the package.json to the package folder? This will copy dependencies like Sveltkit itself that is really not needed within the package. 

Is there a way to disable this behaviour?

**Comment by antony** [comment]
Dependencies such as SvelteKit should be declared as dev dependencies, which your install process should not be installing in a production environment.

If you're ending up with dev dependencies in your production code, this is an issue with your build process, rather than the package command.
